avaframe/com1DFAPy/com1DFACfg.ini

```diff
@@ -24,6 +24,8 @@ hmin = 0.05
 sphOption = 1
 # minimum SPH distance [m]
 minRKern = 0.001
+# sph kernel smoothing length
+sphKernekRadi = 5
```

Comment:
sphKernelRadius?

Reply:
and maybe add a dimension to it since the smoothing length is also in [m], right?